### ISSUE TITLE
Mori Tiang

### ISSUE THREAD

**Issue by johnbent** [tekinged/missing#347]
Mori Tiang created by jimgeselbracht@yahoo.com on 2021-12-28 15:54:35

**Comment by johnbent** [comment]
jimgeselbracht@yahoo.com replied, <br><br>In the song Tsukihi Ua Nangarete Mo, Tem Obakerbau wrote "Ullebengelel, e ochedengei, le keleng di mori tiang."  Yoichi Rengiil tells me that "mori tiang" means "at this very moment" or "this instant," and is distinct from chelechang in that "mori tiang" means one moment in time while "chelechang" could be today (the whole day).  He used another example sentence to illustrate:  "Kau me ngak a di mori tiang e kede mokebii."  You can hear a recording of Tsukihi below, with the phrase starting at 3:04.  https://hellaholler.com/ouchacha/mp3/044.mp3

**Comment by ChrisPerrette** [comment]
Wouldn't <i>mori tiang</i> just be <i>mo er itiang</i> in this context?

**Comment by Gnalu** [comment]
adi moritiang...singing and talking are the same...there is no pause between each words.
a di mo er tiang becomes dimoritiang. If you listen to songs with mor tiang, there will be a strong emphasis on an "i" before tiang. i dont know why but in older generation, that was like their stamp, my thoughts.

**Comment by smith-371** [comment]
= a di mo er itiang

**Comment by Gnalu** [comment]
tial=tia el
tia-tiang, dont know why but we do love our 'ng's'
(i)tiang was used alot in the older generation, I'll leave it for others to share their thoughts.

**Comment by johnbent** [comment]
So I think this satisfies the explanation from Yoichi that it means this very instance because the i- prefix emphasizes tiang. However, TIANG means HERE and does not mean NOW. Can TIANG also mean NOW or does the song actually mean HERE or is this this another word?

**Comment by Gnalu** [comment]
Now-chelechang
 Chelechang eakdi mo er tiang el melmesumech. Now, right this moment/here/instance, I say goodbye.

**Comment by ChrisPerrette** [comment]
I think <i>tia(ng)</i> can mean <i>this</i> in a wide variety of ways, this thing, this place, this moment/time, etc.

**Comment by Lukes-Imeyuns** [comment]
Yes tiang means HERE or THIS. Tiang el basio, subong, kukau, cheldecheduch,
etc for "This". And when it means "here" it would be, for example: A
ongdibel a di mor ngii er tiang.

However, in that song, and in conversations, when you say ng di mor itiang,
then it's referring to THIS very instant or right now.
